### ISSUE TITLE
Keyboard navigation across the track header controls is broken

### ISSUE THREAD

**Issue by dozzzzer** [audacity/audacity#11180]
1. Add a new track
2. Tab through the elements until you get to the track header
3. Press Enter
4. Press the arrow keys on the keyboard
5. Observe: you can only move focus to either the Effects button or the Track name by pressing either the left or right arrow key, respectively, and from there, you cannot move focus anywhere. 

https://github.com/user-attachments/assets/a1cdad7f-5ed4-4342-8a28-1335ac2a7913

regression since bd971ed
